pkg/operator/k8s/pod.go

```diff
@@ -40,6 +40,7 @@ const (
 	PodStatusFailed      = "Failed"
 	PodStatusKilled      = "Killed"
 	PodStatusUnknown     = "Unknown"
```

Comment:
I do you think Unknown should be last in the list?

Reply:
I think it could be convenient if we refactor this into enums